_i18n/en.yml

```diff
@@ -279,8 +284,87 @@ help_pages:
       **Read more**
       [Phone didn’t get a security code](site.baseurl/help/creating-an-account/phone-didnt-get-a-security-code/)
       [No phone or other authentication method](site.baseurl/help/creating-an-account/no-phone-or-other-authentication-method/)
+  manage-your-account:
+    overview: |-
+      # Manage your account
+      Manage your account settings including your password, phone number, email, and more.
+
+      ## Common troubleshooting topics
+      * [Change your password](#)
+      * [Change the phone number associated with your account](#)
+      * [Change the email address associated with your account](#)
+      * [Delete your login.gov account](#)
+      * [Manage multiple login.gov accounts](#)
```

Comment:
These links don't go anywhere, is the goal to have them here in the i18n YML or to generate them in the template like we do with the navbar? Maybe we could add a `{{ help_index }}` variable that we replace out with with content?

Reply:
Yup, they're placeholders. I'm building the functionality in a [separate branch](https://github.com/18F/identity-site/pull/505). The idea is to loop through the category list if the `show_common_issues` variable is true. I didn't use `{{ help_index }}` but please feel free to build on what I have in the `ca-build-common-help-issues` branch.